Resources/public/js/models/ez-contentmodel.js

```diff
@@ -232,45 +232,69 @@ YUI.add('ez-contentmodel', function (Y) {
          * @param {Function} callback
          */
         loadLocations: function (options, callback) {
-            var api = options.api,
-                currentLocation = options.location;
+            var locations = [],
+                that = this;
 
-            api.getContentService().loadLocations(this.get('id'), function (error, response) {
-                var tasks = new Y.Parallel(),
-                    loadError = false,
-                    locations = [];
+            this._loadLocationsView(options, function (error, response) {
```

Comment:
if you keep this method, please bind the callback to `this` and remove the ugly `that = this`